Packs/CybleEventsV2/Integrations/CybleEventsV2/CybleEventsV2.yml

```diff
@@ -151,28 +161,33 @@ script:
     - defaultValue: '5'
       description: Number of records to return (max 50). Using a smaller limit will get faster responses.
       name: limit
-    - description: Timeline start date in the format "%Y-%m-%dT%H:%M:%S%z" (iso-8601)
+    - description: Timeline start date in the format "%Y-%m-%dT%H:%M:%S%z" (iso-8601).
       name: start_date
       required: true
-    - description: Timeline end date in the format "%Y-%m-%dT%H:%M:%S%z"  (iso-8601)
+    - description: Timeline end date in the format "%Y-%m-%dT%H:%M:%S%z"  (iso-8601).
       name: end_date
       required: true
     - defaultValue: '0'
       description: Returns records for the timeline starting from the given indice.
       name: from
       required: true
-    description: Fetch incident event group
+    description: Fetch incident event group.
     name: cyble-vision-fetch-alert-groups
     outputs:
     - contextPath: CybleEvents.AlertGroup
-      description: Fetch all the alert groups
+      description: Fetch all the alert groups.
       type: String
-  dockerimage: demisto/python3:3.10.12.63474
+  - description: Retrieves a User Profile schema, which holds all of the user fields within the application. Used for outgoing-mapping through the Get Schema option.
+    name: get-mapping-fields
+  dockerimage: demisto/python3:3.10.13.73190
   isfetch: true
   runonce: false
   script: '-'
   subtype: python3
   type: python
+  ismappable: true
+  isremotesyncin: true
```

Comment:
isn't this only for mapping in? you added only outgoing mapper. is that correct?

Reply:
Yes, currently only the outgoing mapping is functional. Though I have added `isremotesyncin: true` too, soon we will be adding bi-directional mapping as well.